src/renderer/shared/api/translation/locales/en.json

```diff
@@ -102,6 +102,8 @@
     },
     "searchPlaceholder": "Search",
     "selectAll": "Select all",
+    "selectedStatus_one": "{ selected } out of { count } operation selected",
+    "selectedStatus_other": "{ selected } out of { count } operations selected",
```

Comment:
Where exactly are we using them? I see only `basket.selectedStatus` there

Reply:
it's i18next [plurals feature](https://www.i18next.com/translation-function/plurals), it use `_one` suffix if `count` is 1, and `_other` if other